docs/python/python-tutorial.md

```diff
@@ -47,13 +47,13 @@ IntelliSense and auto-completions work for standard Python modules as well as ot
 
 ![IntelliSense appearing for a variable whose type provides methods](images/tutorial/intellisense02.png)
 
-Feel free to experiment with IntelliSense some more, but then revert changes so you have only the `msg` variable and the `print` statement, and save the file (`kb(workbench.action.files.save)`).
+Feel free to experiment with IntelliSense some more, but then revert changes so you have only the `msg` variable and the `print` call, and save the file (`kb(workbench.action.files.save)`).
 
 For full details on editing, formatting, and refactoring, see [Editing code](/docs/python/editing.md). The Python extension also has full support for [Linting](/docs/python/linting.md).
 
 ## Running Hello World
 
-It's simple to run `hello.py` with Python. From an external terminal, just type `python hello.py`, and you should see "Hello World" as output.
+It's simple to run `hello.py` with Python. From an external terminal, just type `python3 hello.py`, and you should see "Hello World" as output.
```

Comment:
Why "python3" when the python paths in the settings just use "python"? In my 3.6 installation, it's just "python.exe" not python3.exe:

c:\anaconda3\python.exe
C:\python36-32\python.exe

We could say "(or `python3 hello.py` depending on your installation)" if you're concerned about it.

Reply:
The `python`-only thing is Windows only. On macOS and Linux the only way to have `python` point to Python 3 is to manually tweak it or to be using a virtual environment.

Reply:
Does that not affect settings like pythonPath then? What I'm concerned about is that if you put in 'python3' then it breaks for Windows just as "python" breaks for Linux. So I think we need both if they are different.

Reply:
I should also mention that `python` is not on your PATH on Windows unless you opt into it being there.